jit/v2/JIT_v2.0.0.yaml

```diff
@@ -1664,129 +1565,43 @@ components:
+
+            **Condition:** Can only reply to a **Timestamp** with the same `portCallServiceID`
           example: 0342254a-5927-4856-b9c9-aa12e7c00563
         portCallServiceID:
           type: string
           format: uuid
           description: |
             Links the timestamp to a **Port Call Service**.
           example: 0342254a-5927-4856-b9c9-aa12e7c00563
-        portCallServiceDateTime:
+        classifierCode:
           type: string
-          format: date-time
           description: |
-            The local date and time, when the **Port Call Service** will be provided, in [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) format.
-          example: '2019-11-12T07:41:00+08:30'
-        delayReasonCode:
-          type: string
-          maxLength: 3
-          description: |
-            Reason code for the delay. See SMDG [Code list DELAY](https://smdg.org/documents/smdg-code-lists/delay-reason-and-port-call-activity/) for a list of valid codes to be used for this attribute.
-          example: STR
-        isFYI:
-          type: boolean
-          description: |
-            If set to `true` it indicates that this message is meant for another party - but is sent as a FYI (for your information).
-          example: true
-        remark:
-          type: string
-          maxLength: 500
-          description: |
-            Free text to provide additional information on the context.
-          example: Port closed due to strike
-      required:
-        - timestampID
-        - portCallServiceID
-        - portCallServiceDateTime
+            Code for describing the timestamp.
 
-    #####################
-    # Requested timestamp
-    #####################
-    RequestedTimestamp:
-      title: Requested Timestamp
-      type: object
-      description: |
-        Requested date and time when a **Port Call Service** should be provided.
-      properties:
-        timestampID:
-          type: string
-          format: uuid
-          description: |
-            Universally unique identifier for the **Timestamp**. The creator of the **Timestamp** is responsible to create the `timestampID`
-          example: 0342254a-5927-4856-b9c9-aa12e7c00563
-        replyToTimestampID:
-          type: string
-          format: uuid
-          description: |
-            The identifier of the **Timestamp** being replied to.
-          example: 0342254a-5927-4856-b9c9-aa12e7c00563
-        portCallServiceID:
-          type: string
-          format: uuid
-          description: |
-            Links the timestamp to a **Port Call Service**.
-          example: 0342254a-5927-4856-b9c9-aa12e7c00563
-        portCallServiceDateTime:
+            Possible values are:
+            - `ACT` (Actual)
+            - `EST` (Estimated)
+            - `PLN` (Planned)
+            - `REQ` (Requested)
+
+            **Condition:** `EST`, `PLN` and `ACT` are **only** to be used by the **consumer**.
```

Comment:
The provider of the service is posting EST, PLN, ACT to the consumer of the Service. 

The consumer of the service is posting the REQ to the provider. 

From what angle are you writing it? the implementing party of the endpoint or the one supposed to call that endpoint?

Reply:
From the one that calls the endPoint. The API should describe how to use it - not how it is implemented ;-)

Reply:
Are you suggesting to change the text?